src/objects/texts/zcl_abapgit_lxe_texts.clas.abap

```diff
@@ -0,0 +1,169 @@
+CLASS zcl_abapgit_lxe_texts DEFINITION
+  PUBLIC
+  FINAL
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+    INTERFACES zif_abapgit_lxe_texts.
+
+    CLASS-METHODS:
+      get_lang_iso4
+        IMPORTING
+          iv_src         TYPE spras
+        RETURNING
+          VALUE(rv_iso4) TYPE lxeisolang ,
+      get_lxe_object_list
+        IMPORTING
+          iv_obj_name        TYPE sobj_name
+          iv_object_type     TYPE trobjtype
+        RETURNING
+          VALUE(rt_obj_list) TYPE lxe_tt_colob,
+      get_lxe_texts
+        IMPORTING
+                  iv_original_language TYPE spras
+                  iv_obj_name          TYPE sobj_name
+                  iv_object_type       TYPE trobjtype
+        RETURNING VALUE(rt_lxe_texts)  TYPE zif_abapgit_lxe_texts~ty_tlxe_i18n,
+      deserialize_lxe_texts
+        IMPORTING
+          it_lxe_texts TYPE zif_abapgit_lxe_texts~ty_tlxe_i18n,
+      get_installed_languages
+        RETURNING
+          VALUE(rt_installed_languages) TYPE zif_abapgit_lxe_texts~ty_languages.
+  PROTECTED SECTION.
+  PRIVATE SECTION.
+
+ENDCLASS.
+
+
+
+CLASS zcl_abapgit_lxe_texts IMPLEMENTATION.
+  METHOD get_lang_iso4.
+    CALL FUNCTION 'LXE_T002_CONVERT_2_TO_4'
+      EXPORTING
+        old_r3_lang = iv_src
+      IMPORTING
+        new_lang    = rv_iso4.
+  ENDMETHOD.
+
+
+  METHOD get_lxe_object_list.
+
+
+    DATA lv_object_name TYPE trobj_name.
+
+    lv_object_name = iv_obj_name.
+    CALL FUNCTION 'LXE_OBJ_EXPAND_TRANSPORT_OBJ'
+      EXPORTING
+        pgmid    = 'R3TR'
+        object   = iv_object_type
+        obj_name = lv_object_name
+      TABLES
+        ex_colob = rt_obj_list.
+
+  ENDMETHOD.
+  METHOD get_lxe_texts.
+    DATA:
+      lt_obj_list            TYPE lxe_tt_colob,
+      lt_installed_languages TYPE TABLE OF langu,
+
+      ls_lxe_text_item       TYPE zif_abapgit_lxe_texts~ty_lxe_i18n.
```

Comment:
just a style thing: we prefer to use `=>` to reference types in abapGit. keep `~` for interface methods but replace `~` with `=>` for types

Reply:
Done